src/test/scala/introcourse/level07/LogParserTest.scala

```diff
@@ -1,12 +1,12 @@
 package introcourse.level07
 
-import introcourse.level07.LogParser._
+import introcourse.level07.LogParser.*
```

Comment:
This test expects names like `KnownLog`, `UnknownLog`, `Info`, etc to be in scope, but the change from sealed trait to enum puts these values in their own namespace matching the enum's name. Therefore, to make it easy for learners, I'd suggest we bring those values into scope.

```suggestion
import introcourse.level07.LogParser.*
import introcourse.level07.LogParser.LogLevel.*
import introcourse.level07.LogParser.LogMessage.*
```